docs/docs/agents/agents.md

```diff
@@ -55,63 +55,66 @@ agent = create_react_agent(
 )
 ```
 
+See the [models](./models.md) page for more information on how to configure LLMs.
+
 ## Custom Prompts
 
 Prompts instruct the LLM how to behave. They can be:
 
 * **Static**: A fixed string or list of [messages](https://python.langchain.com/docs/concepts/messages/)
```

Comment:
actually this one is misleading, because you can't currently pass a list of messages as a prompt to `create_react_agent`, so it might confuse people. we probably need to rephrase

Reply:
Got it -- will add to do list

Reply:
updated